backend/benefit/applications/tests/test_applications_api.py

```diff
@@ -128,6 +129,44 @@ def test_applications_filter_by_ssn(api_client, application, association_applica
     assert response.status_code == 200
 
 
+def test_applications_simple_list(handler_api_client, received_application):
+    response = handler_api_client.get(
+        reverse("v1:handler-application-simplified-application-list")
+    )
+    assert len(response.data) == 1
+    for key in BaseApplicationViewSet.EXCLUDE_FIELDS_FROM_SIMPLE_LIST:
+        assert key not in response.data
+    assert response.status_code == 200
+
+
+def test_applications_simple_list_exclude_more(
+    handler_api_client, received_application
+):
+    response = handler_api_client.get(
+        reverse("v1:handler-application-simplified-application-list")
+        + "?exclude_fields=status"
+    )
```

Comment:
Adding a test case for multiple exclude_field values might be good but I leave that up to you to decide whether to add one or not.

Reply:
Added some more tests